i18n/en.json

```diff
@@ -6136,12 +6136,16 @@
     "translation": "An error occurred while saving the file to {{.Filename}}"
   },
   {
+    "id": "utils.config.supported_available_locales.app_error",
+    "translation": "Unable to load mattermost configuration file:  AvailableLocales must include DefaultClientLocale. Set AvailableLocales to empty string as default value."
```

Comment:
Can you change `Set AvailableLocales` to `Setting AvailableLocales` so that it's more clear that the server is doing it automatically? Also, can you change `empty string` to `all locales` since the person setting up the server may not know what `empty string` does here